src/main/kotlin/no/nav/familie/ef/sak/vurdering/DelvilkårRegler.kt

```diff
@@ -0,0 +1,72 @@
+package no.nav.familie.ef.sak.vurdering
+
+import no.nav.familie.ef.sak.api.dto.InngangsvilkårGrunnlagDto
+import no.nav.familie.ef.sak.api.dto.Sivilstandstype
+import no.nav.familie.ef.sak.api.dto.VilkårsvurderingDto
+import no.nav.familie.ef.sak.repository.domain.DelvilkårType
+import no.nav.familie.ef.sak.repository.domain.Vilkårsresultat
+import no.nav.familie.ef.sak.repository.domain.Vilkårsvurdering
+import no.nav.familie.ef.sak.repository.domain.søknad.SøknadsskjemaOvergangsstønad
+
+
+/**
+ * Filtrerer bort delvikår som ikke skall vurderes iht data i søknaden
+ */
+
+fun utledDelvilkårResultat(delvilkårType: DelvilkårType,
+                           søknad: SøknadsskjemaOvergangsstønad,
+                           registerGrunnlag: InngangsvilkårGrunnlagDto): Vilkårsresultat {
+    return if (erDelvilkårAktueltForSøknaden(delvilkårType, søknad, registerGrunnlag)) {
+        Vilkårsresultat.IKKE_VURDERT
+    } else {
+        Vilkårsresultat.IKKE_AKTUELL
+    }
+}
+
+fun validerDelvilkår(vurdering: VilkårsvurderingDto,
+                     vilkårsvurdering: Vilkårsvurdering) {
+    val innkommendeDelvurderinger = vurdering.delvilkårsvurderinger.map { it.type }.toSet()
+    val lagredeDelvurderinger = vilkårsvurdering.delvilkårsvurdering.delvilkårsvurderinger.map { it.type }.toSet()
+
+    if (innkommendeDelvurderinger.size != lagredeDelvurderinger.size
```

Comment:
Bytte til when() -lit tvanskelig å lese if'er. 
evt bytte til to check, eller require?